schema/manager.go

```diff
@@ -43,9 +40,13 @@ type Manager struct {
 	policies    []*Policy
 	Extensions  []*Extension
 	namespaces  map[string]*Namespace
+	mu          sync.RWMutex
```

Comment:
This is a golang wide naming convention, see in vendor and stdlib. BTW `grep -i` for mutex shows 2 variables